tm2py/components/network/highway/highway_assign.py

```diff
@@ -133,7 +133,7 @@ def run(self):
         run_dynamic_toll = self.config.tolls.run_dynamic_toll
         valuetoll_start_tollbooth_code = self.config.tolls.valuetoll_start_tollbooth_code
         max_dynamic_valuetoll = self.config.tolls.max_dynamic_valuetoll
-        apply_msa = self.config.msa.apply_msa
+        warmstart = self.config.run.warmstart.warmstart
```

Comment:
This does not look correct. Shouldn't it be `self.controller.config.run.warmstart.warmstart`?

Reply:
oh, I have this fixed, but did not push it up here. updated in 7921a19